<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
border: 2px solid  rgb(0, 255, 0);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="TitleBar" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="body" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="videoWidget" native="true">
        <widget class="ControlBar" name="controlBar" native="true">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>540</y>
           <width>631</width>
           <height>50</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
        <widget class="SliderVideo" name="slider_video">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>510</y>
           <width>631</width>
           <height>22</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="Sidebar" name="sidebar" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>251</width>
           <height>491</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:10px;</string>
         </property>
         <widget class="QTreeView" name="dirtree">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>231</width>
            <height>431</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:10px;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="opendir">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>450</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 0);</string>
          </property>
          <property name="text">
           <string>打开</string>
          </property>
         </widget>
        </widget>
        <widget class="PlayImage" name="playImage" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1021</width>
           <height>661</height>
          </rect>
         </property>
         <widget class="QPushButton" name="openfile">
          <property name="geometry">
           <rect>
            <x>530</x>
            <y>240</y>
            <width>91</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <zorder>playImage</zorder>
        <zorder>controlBar</zorder>
        <zorder>slider_video</zorder>
        <zorder>sidebar</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>titlebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlBar</class>
   <extends>QWidget</extends>
   <header>controlbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Sidebar</class>
   <extends>QWidget</extends>
   <header>sidebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SliderVideo</class>
   <extends>QSlider</extends>
   <header>slidervideo.h</header>
  </customwidget>
  <customwidget>
   <class>PlayImage</class>
   <extends>QWidget</extends>
   <header>playimage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
